scripts/core/sampler.yml

```diff
@@ -107,6 +107,42 @@ returns:
     - $X_RESULT_ERROR_OUT_OF_RESOURCES
 --- #--------------------------------------------------------------------------
 type: function
+desc: "Create a sampler object in a context"
+class: $xSampler
+name: CreateExp
+ordinal: "0"
+details:
+    - "The props parameter specifies a list of sampler property names and their corresponding values."
+    - "The list is terminated with 0. If the list is NULL, default values will be used."
+analogue:
+    - "**clCreateSamplerWithProperties**"
+params:
+    - type: $x_context_handle_t
+      name: hContext
+      desc: "[in] handle of the context object"
+    - type: const $x_sampler_desc_t*
+      name: pDesc
+      desc: "[in] pointer to the sampler description"
+    - type: float
+      name: minMipmapLevelClamp
+      desc: "[in] minimum mipmap level from which we can sample, minimum value being 0"
+    - type: float
+      name: maxMipmapLevelClamp
+      desc: "[in] maximum mipmap level from which we can sample, maximum value being the number of levels"
+    - type: float
+      name: maxAnistropy
+      desc: "[in] anisotropic ratio used when samplling the mipmap with anisotropic filtering"
```

Comment:
I don't think we need a new `urSamplerCreateExp` entry point in order to specify these extra arguments because the `pDesc` argument is extensible using a `pNext` chain.

For an example of this pattern take a look at [urMemBufferCreate](https://oneapi-src.github.io/unified-runtime/core/api.html#urmembuffercreate) where the [ur_buffer_channel_properties_t](https://oneapi-src.github.io/unified-runtime/core/api.html#structur__buffer__channel__properties__t) and [ur_buffer_alloc_location_properties_t](https://oneapi-src.github.io/unified-runtime/core/api.html#structur__buffer__alloc__location__properties__t) structs can be optionally specificed as part of a `pNext` chain.

This new struct should also be defined in the `source/core/exp-bindless-images.yml` file leaving no changes to this file.

Reply:
Done. The new struct is `$x_exp_sampler_mip_properties_t` in `source/core/exp-bindless-images.yml`